java/test/model/game/GameTest.java

```diff
@@ -463,5 +465,14 @@ public void testCanBuildRoadandCity() throws InvalidPlayerException, PlayerExist
             assertTrue(game.canBuildCity(current_turn,vloc));
         }
 
+        int settlements_available_before = game.getAvailableSettlements(current_turn);
+        int cities_available_before = game.getAvailableCities(current_turn);
+        game.buildCity(current_turn,vloc);
+        int settlements_available_after = game.getAvailableSettlements(current_turn);
+        int cities_available_after = game.getAvailableCities(current_turn);
+
+        assertTrue(settlements_available_after == settlements_available_before+1);
+        assertTrue(cities_available_after == cities_available_before-1);
```

Comment:
These assertions should be
```java
assertEqual(settlements_available_after, settlements_available_before+1);
assertEqual(cities_available_after, cities_available_before-1);
```
since you're checking for equality with two values, not evaluating a boolean as `assertTrue` is intended for.